pkg/sfu/buffer/buffer.go

```diff
@@ -457,14 +457,14 @@ func (b *Buffer) SetPLIThrottle(duration int64) {
 
 func (b *Buffer) SendPLI(force bool) {
 	b.RLock()
-	if (b.rtpStats == nil || b.rtpStats.TimeSinceLastPli() < b.pliThrottle) && !force {
-		b.RUnlock()
+	rtpStats := b.rtpStats
```

Comment:
Can we change the RLock to Lock to fix the race condition?

Reply:
Yeah `Lock` would work. I think it is fine to push it out to rtpstats also.

Reply:
that would fix it, I thought it would be better to not hold the lock since rtpStats can lock